src/main.js

```diff
@@ -327,12 +327,21 @@ ready(() => {
                     fileName = promptReply;
                 }
 
-                createNewSubmission(id, fileName, (res) => {
-                    console.log("Created submission, return data", res)
-                    var submissionId = res.data.submission.submissionId;
```

Comment:
nit: const or let, especially because another `submissionId` is enclosed further down the scope on 332. I know I haven't been clean on this either lol.